src/openrct2/ride/TrackData.cpp

```diff
@@ -698,6 +700,8 @@ static constexpr uint8_t TrackSequenceProperties[][MaxSequencesPerPiece] = {
     { 0 }, // TrackElemType::RightEighthBankToOrthogonalUp25 
     { 0 }, // TrackElemType::LeftEighthBankToOrthogonalDown25
     { 0 }, // TrackElemType::RightEighthBankToOrthogonalDown25
+    /* DiagonalBrakes                        */ { 0 },
```

Comment:
Make this match the style of the others next to this.

Reply:
The way I did it is the style that TrackSequenceProperties originally used. The style with the track name after was introduced by X7123M3-256 for his track pieces. Is that the way you want them all to be?

Reply:
Argh... hmm keep as is then.